src/aap_eda/core/models/mixins.py

```diff
@@ -0,0 +1,21 @@
+from django.utils import timezone
+
+
+class ModifiedAtUpdaterMixin:
+    """Mixin to ensure that the `modified_at` field is updated.
+
+    Ensure that the `modified_at` field is updated when the model
+    is saved with specific fields (`update_fields`) specified for update.
+    Only works for `auto_now` fields.
+    """
+
+    def save(self, *args, **kwargs):
+        for field in self._meta.fields:
```

Comment:
Rather than iterating the code could use `self._meta.get_field("modified_at")`; would need to catch the `FieldDoesNotExist` exception.

Reply:
Good idea! thanks

Reply:
updated.